README.md

```diff
@@ -48,6 +54,33 @@ python3 -m unittest
 ```
 
 
+### How to publish a new version
```

Comment:
I think this is more internal doc than something anyone would need on a README. If you really want it, move it in a Wiki page of the github repo.

Reply:
I would prefer to be able to manage it through the code editor. What do you think about `PUBLISH.md`?

Reply:
Sure, is way better :)